### DISCUSSION TITLE
JSPI default=ON ?

### DISCUSSION THREAD

**Discussion by rentallect** [emscripten-core/emscripten#20400] [Q&A] [answered]
Does anyone know when the "experimental" JSPI feature will be enabled by default in Chrome/Brave/Edge?

**Comment by sbc100** [comment, marked as answer]
Its currently at phase 3: https://github.com/WebAssembly/proposals.

The browsers likely won't enable it by default until it reaches phase 4.

You can follow the progress either in the propsals repo or in the specific repo for this proposal: https://github.com/WebAssembly/js-promise-integration/.
